extension/content.js

```diff
@@ -148,11 +148,12 @@ function addDeleteForkLink() {
 function linkifyIssuesInTitles() {
 	const $title = $('.js-issue-title');
 	const titleText = $title.text();
+    const issueRegex = utils.issueRegex;
 
-	if (/(#\d+)/.test(titleText)) {
+	if (issueRegex.test(titleText)) {
 		$title.html(titleText.replace(
-			/#(\d+)/g,
-			`<a href="https://github.com/${repoUrl}/issues/$1">#$1</a>`
+			RegExp(issueRegex.source, 'g'),
```

Comment:
Not semantically. Just a preference and for consistency. You always use `new` for your own constructors. `RegExp` without `new` is just a quirk in the language.